request-manager/grapher/grapher_test.go

```diff
@@ -549,6 +561,144 @@ func TestFailCreateBadIfConditionalGraph(t *testing.T) {
 	}
 }
 
+func TestLimitParallel(t *testing.T) {
+	tf := &testFactory{}
+	sequencesFile := "../test/specs/decomm-limit-parallel.yaml"
+	cfg, _ := ReadConfig(sequencesFile)
+	omg := NewGrapher(req, tf, cfg, id.NewGenerator(4, 100))
```

Comment:
looks like the other tests all use [funcs like this ](https://github.com/square/spincycle/blob/c30fa50b0f7ce5804aedb1bb02b10d662348b8a8/request-manager/grapher/grapher_test.go#L81-L93)to make their test graphers. For consistency's sake, can you do the same thing here?